.github/dependabot.yml

```diff
@@ -18,6 +18,28 @@ updates:
     schedule:
       interval: "daily"
     groups:
-      pip:
+      action:
         patterns:
-          - '*'
+          - "clang-tools*"
+          - "cpp-linter*"
+  - package-ecosystem: uv
+    directory: /
+    schedule:
+      interval: "quarterly"
+    groups:
+      dev:
+        patterns:
+          - "mypy*"
+          - "pre-commit*"
+          - "ruff*"
+        update-types:
+          - "version-update:semver-major"
+          - "version-update:semver-minor"
```

Comment:
Please note the supported values in [dependabot docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#update-types-groups).